src/sip.c

```diff
@@ -598,15 +619,13 @@ sip_parse_msg_media(sip_msg_t *msg, const u_char *payload)
                      * and port of the stream.
                      */
                     // Create a new stream with this destination address:port
-                    if (!call_msg_is_retrans(msg)) {
-                        if (!rtp_find_call_stream(call, src, dst)) {
-                            // Create RTP stream
-                            call_add_stream(call, stream_create(media, dst, PACKET_RTP));
-                            // Create early RTCP stream
-                            dst.port++;
-                            call_add_stream(call, stream_create(media, dst, PACKET_RTCP));
-                        }
-                    }
+
+                    // Create RTP stream
+                    rtp_stream = stream_create(media, dst, PACKET_RTP);
+
+                    // Create RTCP stream
+                    rtcp_stream = stream_create(media, dst, PACKET_RTCP);
+                    rtcp_stream->dst.port++;
                 }
             }
         }
```

Comment:
Ok, maybe I must implement a way to make retransmissions have the same media list that the original message, so arrows still print their SDP like this:
![retrans](https://cloud.githubusercontent.com/assets/800069/17296112/dbc6eb50-57ff-11e6-82a6-25bd8188abe1.png)